pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -44,6 +49,24 @@ const (
 	taskRunControllerName = "TaskRun"
 )
 
+const (
+	descScheduling       = "scheduling"
+	descInitializing     = "initializing"
+	descRunning          = "running"
+	descSucceeded        = "succeeded"
+	descFailed           = "failed"
+	descUnknown          = "unknown status"
+	descMissingCondition = "missing end condition"
+)
+
+var (
+	groupVersionKind = schema.GroupVersionKind{
```

Comment:
- [ ] this is misc cleanup but we should change `schema` to `scheme` maybe? kinda weird that we are using both `schema` and `scheme`

Reply:
I'm not sure what is suggested here, the lib's default name is `schema`.  Would you want `schema` to be aliased to something else?

Reply:
nope my bad, i misunderstood and thought _we_ had made a package called `schema`